src/core/src/bootstrap/Constants.py

```diff
@@ -261,8 +261,12 @@ class PackageClassification(EnumBackport):
         'IfRequired': REBOOT_IF_REQUIRED,
         'Always': REBOOT_ALWAYS
     }
-    REBOOT_BUFFER_IN_MINUTES = 15
-    REBOOT_WAIT_TIMEOUT_IN_MINUTES = 5
+    REBOOT_BUFFER_IN_MINUTES = 15                # minimum MW time required to consider rebooting if required (notify - 3, wait - 7, machine - 5)
+    REBOOT_NOTIFY_TIMEOUT_IN_MINUTES = 3         # time to broadcast reboot notification to other processes
```

Comment:
This reads as it is the timeout for notification but is also used as time to reboot or shutdown. Should be renamed to REBOOT_IN_MINUTES or REBOOT_VM_IN_MINUTES?

Reply:
Updated to this -- REBOOT_NOTIFY_WINDOW_IN_MINUTES = 3          # time to broadcast reboot notification to other processes as part of the reboot command

It's definitely not a timeout so removed that. Updated the comment to explain where it's used.